packages/app-store/zapier/api/subscriptions/addSubscription.ts

```diff
@@ -1,26 +1,41 @@
 import type { NextApiRequest, NextApiResponse } from "next";
 
+import isAuthorized from "@calcom/features/auth/lib/oAuthAuthorization";
 import findValidApiKey from "@calcom/features/ee/api-keys/lib/findValidApiKey";
 import { addSubscription } from "@calcom/features/webhooks/lib/scheduleTrigger";
 import { defaultHandler, defaultResponder } from "@calcom/lib/server";
 
 async function handler(req: NextApiRequest, res: NextApiResponse) {
   const apiKey = req.query.apiKey as string;
 
-  if (!apiKey) {
-    return res.status(401).json({ message: "No API key provided" });
+  let validKey: any = null;
```

Comment:
Havent tested code still performs the same way but i think we can remove the any and null define here. But on a type level we can improve this 

```suggestion
  let validKey;
```
![CleanShot 2023-09-26 at 12 21 01](https://github.com/calcom/cal.com/assets/55134778/25d1aebb-36c6-453b-ba20-eae0ea37fc70)

Reply:
If we need this to be null and not undefined ignore this